WireAuthentication/Sources/WireAuthenticationUI/Views/Registration/VerificationEmailCode/VerificationEmailCodeView.swift

```diff
@@ -0,0 +1,111 @@
+//
+// Wire
+// Copyright (C) 2025 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import SwiftUI
+import WireAuthenticationAPI
+import WireDesign
+
+package struct VerificationEmailCodeView: View {
+
+    @StateObject private var viewModel: VerificationEmailCodeViewModel
+    @FocusState private var focusedIndex: Int?
+
+    private typealias Strings = L10n.Localizable
+
+    package init(
+        factory: @autoclosure @escaping () -> VerificationEmailCodeFactory
+    ) {
+        self._viewModel = StateObject(wrappedValue: factory().viewModel)
+    }
+
+    package var body: some View {
+        VStack(spacing: 20) {
+            Text(Strings.VerificationCode.message("viewModel.email"))
+                .wireTextStyle(.body1)
+                .multilineTextAlignment(.center)
+                .foregroundStyle(Color.primaryText)
+
+            verificationCodeView
+
+            Button(action: {
+                Task { await viewModel.confirm() }
+            }, label: {
+                HStack {
+                    if viewModel.isLoading {
+                        ProgressView()
+                    }
+
+                    Text(Strings.VerificationCode.confirm)
+                }
+            })
+            .wireButtonStyle(.primary)
+            .padding(.horizontal)
+            .disabled(viewModel.isConfirmButtonDisabled)
+
+            Button(action: {
+                Task.detached { await viewModel.requestVerificationCode() }
+            }, label: {
+                Text(Strings.VerificationCode.resendCode)
+            })
+            .wireButtonStyle(.link)
+            .disabled(viewModel.isResending)
+        }
+        .padding()
+        .background(ColorTheme.Backgrounds.surface.color)
+        .navigationTitle(Strings.VerificationCode.title)
+        .navigationBarTitleDisplayMode(.inline)
+        .setPreferredSize(navigationBarHidden: false)
+        .customBackButton()
+        .alert(
+            item: $viewModel.alert,
+            title: { Text($0.title) },
+            message: { Text($0.message) },
+            actions: { _ in
+                Button(Strings.Authentication.Error.confirm, action: {})
```

Comment:
question: is this closure supposed to be empty? If not maybe a comment or fatalError could be added.

Reply:
yes, we should just close the alert